airbyte-integrations/connectors/source-recurly/source_recurly/streams.py

```diff
@@ -0,0 +1,219 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from abc import abstractmethod
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Union
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import Stream
+from recurly import Client
+
+
+class BaseRecurlyStream(Stream):
+    DEFAULT_PRIMARY_KEY = "id"
+    DEFAULT_CURSOR = "updated_at"
+    DEFAULT_SORT_KEY = "updated_at"
+
+    BEGIN_TIME_PARAM = "begin_time"
```

Comment:
Look other connectors how they set those constant/variables.

Reply:
Fixed in https://github.com/airbytehq/airbyte/pull/8675/commits/5b90ba6758fa44c555bb9a28acafc4c80f22904b